extensions/BugModal/web/autosize.min.js

```diff
@@ -1,6 +1,6 @@
 /*!
-	Autosize 3.0.20
+	Autosize 4.0.0
```

Comment:
The commit message says "Requires a tweaked version of the autosize library." However, no tweaks are documented in the comments here, nor is authorship updated below. Where is the commit reflecting the tweaks in pre-minimized form? How can we regenerate autosize.min.js including your tweaks if we need to (for example) merge Autosize 4.0.1 someday?